rpcserver.go

```diff
@@ -4508,6 +4508,18 @@ func (r *rpcServer) DescribeGraph(ctx context.Context,
 	// transactional model.
 	graph := r.server.chanDB.ChannelGraph()
 
+	// Calculate betweenness centrality if requested. Note that depending on the
+	// graph size, this may take up to a few minutes.
+	var centrality map[autopilot.NodeID]float64
+	if req.CalculateNodeCentrality {
+		channelGraph := autopilot.ChannelGraphFromDatabase(graph)
+		centralityMetric := autopilot.NewBetweennessCentralityMetric()
+		if err := centralityMetric.Refresh(channelGraph); err != nil {
```

Comment:
do we ever use the `refresh` functionality, or could we just pass the channel graph into the contructor above? (or `GetMetric`)

Reply:
we don't use it for this metric as this algo requires full recalculation, but in general i'd say it's better to be able to refresh rather then calculate once upon construction as there may be partial result that is cheaper to update. wdyt?

Reply:
I think there's a lot to that argument, but maybe do that change where we have a metric that actually requires this. The interface can easily be changed later.

Reply:
I'd like to keep it this way for now. As we don't really have information right now whether or not the graph is dirty it makes sense to separate calculation from result retrieval. Once this is wired into autopilot, recalculation may happen less frequently then retrieval based on user preference.